fail2ban/server/filter.py

```diff
@@ -518,19 +526,45 @@ def findFailure(self, tupleLine, date=None, returnRawHost=False,
 						 % ("\n".join(failRegex.getMatchedLines()), timeText))
 				else:
 					self.__lineBuffer = failRegex.getUnmatchedTupleLines()
+					# retrieve failure-id, host, etc from failure match:
+					raw = returnRawHost
```

Comment:
raw is exactly that: it should return failure-id, without checking it by resolve dns etc.
It is just:
- backwards compatibility to the current master, see the workaround in #1454 (using not new `fid` but `host`);
- raw handling in fail2ban-regex command line;
- possibility to add a host as failure (to put in raw in some special action, that can ban host names);